plugins/giffun.js

```
@@ -0,0 +1,538 @@
+}
+
+plugin({
+    pattern: 'cry ?(.*)',
+    desc: 'Send a crying reaction GIF.',
+    react: "😢",
+    type: "fun",
+    fromMe: mode
+}, async (message, match) => {
+    try {
+        let sender = `@${message.sender.split("@")[0]}`;
+        let mentionedUser = message.mention[0] || (message.reply_message && message.reply_message.sender);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)